src/Tutor.Core/LearnerModel/DomainOverlay/KnowledgeComponentMasteries/KnowledgeComponentMastery.cs

```diff
@@ -22,11 +22,12 @@ public class KnowledgeComponentMastery : EventSourcedAggregateRoot
     public KnowledgeComponent KnowledgeComponent { get; private set; }
     public List<AssessmentItemMastery> AssessmentMasteries { get; private set; }
     public double Mastery { get; private set; }
-    public bool HasActiveSession { get; private set; }
     public bool IsStarted { get; private set; }
     public bool IsPassed { get; private set; }
     public bool IsSatisfied { get; private set; }
     public bool IsCompleted { get; private set; }
+    public SessionTracker SessionTracker { get; private set; }
+    public bool HasActiveSession { get; private set; }
```

Comment:
The HasActiveSession property needs to be removed, but to do that, all the seeding scripts need to be corrected. Is there a way to do that without fixing all the inserts for KCM manually?

Reply:
Removed HasActiveSession and fixed scripts in d082c4384bf7a8ad1a6e3c772af2103fa477decc.